proto/internal/temporal/server/api/cluster/v1/message.proto

```diff
@@ -39,3 +44,13 @@ message MembershipInfo {
     repeated string reachable_members = 2;
     repeated RingInfo rings = 3;
 }
+
+message ClusterMember {
+    temporal.server.api.enums.v1.ClusterMemberRole role = 1;
+    string host_id = 2;
+    string rpc_address = 3;
+    int32 rpc_port = 4;
+    google.protobuf.Timestamp session_start_time = 5 [(gogoproto.stdtime) = true];
+    google.protobuf.Timestamp last_heartbit_time = 6 [(gogoproto.stdtime) = true];
+    google.protobuf.Timestamp record_expiry_time = 7 [(gogoproto.stdtime) = true];
```

Comment:
are these coming from ringpop?

Reply:
this comes from `cluster_membership` table, not ringpop (as i understand this) https://github.com/temporalio/temporal/blob/590b6eaeb0f183216290ac24249abee04a9716d3/common/persistence/cassandra/cluster_metadata_store.go#L136

Reply:
sample output

```
[
  {
    "role": 2,
    "host_id": "0235beb1-4265-11ec-88f8-50e085dc7c06",
    "rpc_address": "127.0.0.1",
    "rpc_port": 6934,
    "session_start_time": "2021-11-10T20:30:13.829851Z",
    "last_heartbit_time": "2021-11-10T22:49:11.729838Z",
    "record_expiry_time": "2021-11-12T22:49:11.729838Z"
  },
  {
    "role": 3,
    "host_id": "02374d05-4265-11ec-88f8-50e085dc7c06",
    "rpc_address": "127.0.0.1",
    "rpc_port": 6935,
    "session_start_time": "2021-11-10T20:30:13.829859Z",
    "last_heartbit_time": "2021-11-10T22:49:11.729886Z",
    "record_expiry_time": "2021-11-12T22:49:11.729886Z"
  },
  ```